target/linux/mpc85xx/files/arch/powerpc/boot/dts/fsl/p2020rdb.dts

```diff
@@ -0,0 +1,318 @@
+/*
+ * P2020 RDB Device Tree Source
+ *
+ * Copyright 2009-2012 Freescale Semiconductor Inc.
```

Comment:
It looks like this dts is mostly from the upstream p2020rdb.dts by the same name isn't it?

I think you could either make a patch with your integration changes, or use the p2020rdb.dts as a template and do your customizations on top of it in a separate new p2020rdb-owrt.dts (or with some other name). Freescale did it upstream with the "p1020rdb-pc_camp_core0.dts" that does some customization on top of the "p1020rdb-pc.dts". So you could just follow that.

Reply:
It's was changed dts from upstream. I change it to patch. That allow to follow upstream changes.